transport_nantes/press/views.py

```diff
@@ -296,3 +297,30 @@ def get_image_extension(file_contents):
         press_mention.og_image_url.save(image_filename, files.File(image_fp))
         press_mention.save()
         return None
+
+
+@csrf_protect
+@permission_required('press.press-editor')
+def check_for_duplicate(request, *args, **kwargs):
+    """Check if the provided URL already exists in PressMention."""
+    url = request.GET.get("url")
+    if url:
+        # Article_link has a unique constraint, so first will return
+        # None if there is no match, and the correct PressMention
+        # object if there is a match.
+        # We use first() rather than get() because get() will raise
+        # an exception if there is no match, and we don't want that.
+        press_mention = PressMention.objects.filter(article_link=url).first()
```

Comment:
But `first()` communicates that you have specified an order and expect multiple.

Use `get()` and handle the exception.